File: Codeforces/Round 666/a.cpp
#include<bits/stdc++.h>

using namespace std;

void solve() {
    int n;
    cin >> n;
    vector<string> a(n);
    for(auto &i: a) {
        cin >> i;
    }
    vector<int> cnt(26, 0);
    for(auto r: a) {
        for(auto c: r) {
            ++cnt[c-'a'];
        }
    }
    for(int i = 0; i < 26; ++i) {
        if(cnt[i] % n != 0) {
            cout << "NO\n";
            return;
        }
    }
    cout << "YES\n";
}

int main() {
    #ifndef ONLINE_JUDGE
        freopen("in.txt", "r", stdin);
    #endif
    int t = 1;
    cin >> t;
    while(t--) {
        solve();
    }
    return 0;
}

File: Codeforces/Round 666/a.go
package main

import (
	"fmt"
)

func solve() {
	var n int
	fmt.Scanf("%d\n", &n)
	s := make([]string, n)
	cnt := make([]int, 26)

	for i := range s {
		fmt.Scanf("%s\n", &s[i])
		for _, c := range s[i] {
			cnt[c - 'a']++
		}
	}
	for _, v := range cnt {
		if v % n != 0 {
			fmt.Println("NO")
			return
		}
	}
	fmt.Println("YES")
}

func main() {
	var t int
	fmt.Scanf("%d\n", &t)
	for t > 0 {
		solve()
		t--
	}
}